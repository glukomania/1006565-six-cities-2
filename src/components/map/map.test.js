import renderer from 'react-test-renderer';
import Map from './map';

it(`Map component is displayed correctly`, () => {
  const props = [{coords: [52.3709553943508, 4.919309666406198]}];

  const map = renderer
    .create(<Map offers={props}/>, {
      createNodeMock: (element) => {
        if (element.type === `div`) {
          return document.createElement(`div`);
        }
        return null;
      }})
    .toJSON();
  expect(map).toMatchSnapshot();
});
