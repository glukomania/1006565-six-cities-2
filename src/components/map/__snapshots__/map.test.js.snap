// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map component is displayed correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
